src/v1/views/shared/Header.ts

```diff
@@ -185,14 +155,7 @@ export default class Header extends View {
 
   // Hide the beacon on primary auth failure. On primary auth success, setBeacon does this job.
   removeLoadingBeacon() {
```

Comment:
this function is also called in by the listener on line 72 (event: `removeLoading`)
With this changes, this function no longer calls `Animations.implode`. Can we test that when that listener triggers, the beacon still looks good?

Reply:
I think that it will. 
`this.setBeacon(null, null)` will evaluate transition to `swap` or `unload`. 
in case of `unload` we will execute `Animations.implode(container)` on line # 135
in case of `swap` we will execute `Animations.swapBeacons` which internally will execute `implode` and `explode`

Reply:
I see. Ok, sounds good. Can we just test it manually please? I think we have tests, but just want to double check